javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $contact = document.querySelector('#contact-form');
+
+function preDefault(e) {
+  e.preventDefault();
+  var messgaeData = {
+    name: document.getElementById('contact-form').elements[0].value,
+    email: document.getElementById('contact-form').elements[1].value,
+    message: document.getElementById('contact-form').elements[2].value
```

Comment:
Instead of querying the dom multiple times, just use your `$contact` variable. This is adding an extra performance cost for no reason

Reply:
i aint even going to lie, i forgot i had that variable